dawn-1/sequencer/full-node-values.yaml

```diff
@@ -0,0 +1,167 @@
+global:
+  namespace: "" # add namespace here
+  useTTY: true
+
+# Override value example for full-node from main chart
+moniker: '' # add moniker here
+
+secretProvider:
+  enabled: false
+
+cometbft:
+  config:
+    p2p:
+      # List of nodes to keep persistent connections to
+      persistentPeers:
+        - fa453f8fa2e916025e5718bc53af97e9089058a8@35.235.126.231:26656
+        - 48d4eb0607a7b27f1af0288e60ac5a4ee6823adc@34.94.126.233:26656
+        - 0aaf5845fa6733b519c73644e17e6afcc53bd4a0@35.236.112.124:26656
+        - 59bc2e293f36900f403c0bfc5d00cd016c5397c3@34.94.60.57:26656
+
+  secrets: # TODO: Add secrets, recommended not to use devContent in prodcution environments
```

Comment:
Is this something we know how to do and is it fine to have `devContent` for a potentially longer running testnet?

Reply:
Yeah we are running we actual secrets in our nodes.